README.md

```diff
@@ -126,13 +147,20 @@ kafkaSource.properties.sasl.mechanism=
 kafkaSource.properties.sasl.jaas.config=
 ```
 ```
-#Spark yarn sink properties. Properties used to deploy and run Spark job in Yarn. Not all are required. Adjust according to your use case.
-spark.submit.thread.pool.size=
-sparkYarnSink.hadoopResourceManagerUrlBase=
+# Recurring sensor properties.
+recurring-sensor.maxJobsPerDuration=8
+recurring-sensor.duration=1h
+```
+```
+#Spark properties. Properties used to deploy and run Spark job. Not all are required. Adjust according to your use case.
+#Where spark jobs will be executed. Available options: yarn, emr.
+spark.submitApi=
+
+#Submit api = YARN
+sparkYarnSink.submitTimeout=160000
 sparkYarnSink.hadoopConfDir=
 sparkYarnSink.sparkHome=
```

Comment:
In `application.properties`, we give a value (probably a suggestion) for `sparkYarnSink.hadoopConfDir` and `sparkYarnSink.sparkHome` (`/opt/hadoop` and `/opt/spark`). Should we also add it here, or remove it in `application.properties` or we deliberately keep a difference?

Reply:
I added it into readme as well